DataProducts/test/lassipatanen/Job/JobPosting.json

```diff
@@ -0,0 +1,201 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Job Posting",
+    "description": "Data product for Job Posting",
+    "version": "1.0"
+  },
+  "paths": {
+    "/test/Job/JobPosting": {
```

Comment:
This path should match the path of the definition file, i.e. `/test/lassipatanen/Job/JobPosting`.

Reply:
I will change this